rpc/params/migration.go

```diff
@@ -37,6 +37,7 @@ type MigrationTargetInfo struct {
 	AuthTag         string   `json:"auth-tag"`
 	Password        string   `json:"password,omitempty"`
 	Macaroons       string   `json:"macaroons,omitempty"`
+	Token           string   `json:"token,omitempty"`
```

Comment:
This is set by InitialteMigration() which is called by the migration CLI. 
We'll need to bump the facade version etc.
Where will the jwt be sourced from?